pkg/goDB/storage/storage.go

```diff
@@ -1,18 +1,15 @@
 package storage
 
 import (
-	"sort"
-	"time"
-
 	"github.com/els0r/goProbe/pkg/goDB/encoder/encoders"
 )
 
 // Block denotes a block of goprobe data
 type Block struct {
-	EncoderType encoders.Type `json:"e,omitempty"`
 	Offset      int64         `json:"p,omitempty"`
```

Comment:
Coming to think of this: why don't we make this more readable to the user? E.g.: `encoder_type` for the json tag, etc.

It's not like this is serialized into json by default and we have to worry about disk space.

Reply:
Then let's drop the manual `json` tag altogether and let Go autogenerate them from the variable name. As you point out: it's not that we have to cover special (de)serialization consistency here.